pyhik/hikvision.py

```diff
@@ -430,6 +432,182 @@ def get_device_info(self):
             _LOGGING.error('There was a problem: %s', err)
             return None
 
+    """
+    This function is added by Ayush Pratap Singh (ayushs56@gmail.com)
+    This function is used to get the status of the device
+    """
+
+    def get_device_status(self):
+        """Parse deviceInfo into dictionary."""
+        device_status = {}
+        url = '%s/ISAPI/System/status' % self.root_url
+        using_digest = False
```

Comment:
Resetting the digest flag isn't needed.  It will be set once the initialize function is called.

Reply:
Removed resetting of digest flag